### PR TITLE
Users can add new fields

### PR DESCRIPTION
**Pull request by RD211**
Some small UI changes to the add field screen and connected it to the API call.

### CODE REVIEW

**Comment by RD211** [comment]
Realized I forgot to pull crops from firebase. One sec

**Comment by RD211** [comment]
Now it should be fine

**Comment by paulmis** [comment]
@RD211 could you link relevant issue(s) to this PR?

**Comment by nadinekuo** [comment]
Great work! Nice that you documented relevant widgets and functions 😄

**Review by nadinekuo — APPROVED** [review]
Great work! Nice that you documented relevant widgets and functions 😄

File: src/frontend/lib/views/addfield/widgets/add_field_map.dart [hunk 1]
```diff
@@ -49,27 +34,43 @@ class _AddFieldMapState extends State<AddFieldMap> {
...
                     mapToolbarEnabled: false,
                     initialCameraPosition: mapSettings.cameraPosition,
                     mapType: mapSettings.mapType,
-                    onCameraMove: (camera)=> mapSettings.setCameraPosition(camera),
-                    polygons: Provider.of<NewFieldProvider>(context).geoPoints.isNotEmpty ? {..._polygons,  Provider.of<NewFieldProvider>(context).getPolygon()}: _polygons,
+                    onCameraMove: (camera) => mapSettings.setCameraPosition(camera),
+                    polygons: Provider.of<NewFieldProvider>(context).geoPoints.isNotEmpty
+                        ? {
+                            Provider.of<NewFieldProvider>(context).getPolygon(
+                              fillColor: Colors.green.withOpacity(0.5),
```
Comment (nadinekuo) [inline src/frontend/lib/views/addfield/widgets/add_field_map.dart:49]: I like the green with opacity hehe

**Pull request merged**